htod.dd

```diff
@@ -2,9 +2,8 @@ Ddoc
 
 $(D_S htod,
 
-$(MESSAGE_BOX orange, $(B Warning) - $(D htod) is no longer considered a core tool and maintained
-    by the same people. More modern tools like $(LINK2 https://github.com/jacob-carlborg/dstep, dstep)
-    might work better for your use case.)
+$(MESSAGE_BOX orange, $(B Warning) - `htod` is not developed actively. Alternatives include 
```

Comment:
Please see $(LINK2 https://wiki.dlang.org/Bindings#Binding_generators, Binding generators on the D Wiki) for a list of alternatives.